brewtils/rest/easy_client.py

```diff
@@ -277,12 +277,13 @@ def create_system(self, system):
         return self.client.post_systems(self.parser.serialize_system(system))
 
     @wrap_response(parse_method="parse_system", parse_many=False, default_exc=SaveError)
-    def update_system(self, system_id, new_commands=None, **kwargs):
+    def update_system(self, system_id, new_commands=None, add_instance=None, **kwargs):
```

Comment:
should `add_instance` be part of `**kwargs`?

Reply:
Yeah, and honestly `new_commands` should be too. I'll move `add_instance` for this PR.